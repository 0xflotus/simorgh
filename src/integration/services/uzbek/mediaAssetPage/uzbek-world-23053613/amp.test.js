/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service uzbek
 * @pathname /uzbek/world-23053613
 * @hasRelatedContent true
 */

import runAmpTests from '../../../../pages/mediaAssetPage/ampTests';

describe('AMP uzbek mediaAssetPage', runAmpTests);
