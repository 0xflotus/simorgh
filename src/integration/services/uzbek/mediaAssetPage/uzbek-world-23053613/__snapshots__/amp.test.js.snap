// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP uzbek mediaAssetPage A11y I can see the skip to content link 1`] = `"Саҳифага ўтиш"`;

exports[`AMP uzbek mediaAssetPage AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP uzbek mediaAssetPage AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP uzbek mediaAssetPage Analytics ATI 1`] = `
Object {
  "vars": Object {
    "contentType": "article-media-asset",
    "domain": "test.bbc.co.uk",
    "sections": "Uzbek, Uzbek - Дунё, Uzbek - MAP, Uzbek - Дунё - MAP, Uzbek - News-category",
    "title": "Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston",
    "uid": 50924,
    "virtualReferrer": "\${documentReferrer}",
  },
}
`;

exports[`AMP uzbek mediaAssetPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/uzbek"`;

exports[`AMP uzbek mediaAssetPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. Би-би-си ташқи сайтлар мазмуни учун масъул эмас. Ташқи линкларга бизнинг ёндашувимиз қандайлиги ҳақида маълумотга эга бўлинг"`;

exports[`AMP uzbek mediaAssetPage Header I can see the branding 1`] = `"BBC News, O'zbek"`;

exports[`AMP uzbek mediaAssetPage I can see the headline 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston"`;

exports[`AMP uzbek mediaAssetPage I can see the related content 1`] = `"Марсель полицияси 11 россиялик ишқибозни озод қилди? world фвфвФранция полицияси аввалроқ қўлга олинган 11 нафар россиялик футбол ишқибозини қўйиб юборган. Бу ҳақида Марселдаги Россия бош консули Сергей Молчанов хабар берди world қфвқ.22 июн 2016Видео, 16 iyun kuni jahonda yuz bergan muhim voqealar sharhi Latin.Ha, bugun Angliyaga qarshi maydonga tushajak yulduzi Garet Beyl sharafiga Uelsning shimoliy Bala shahri ma‘muriyati xuddi shunga qaror qilibdi.17 июн 2016"`;

exports[`AMP uzbek mediaAssetPage I can see the timestamp 1`] = `"17 июн 2016"`;

exports[`AMP uzbek mediaAssetPage Media player embed A11y I can read the media player title 1`] = `"Медиа плейер"`;

exports[`AMP uzbek mediaAssetPage Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/uzbek/world-23053613/p01qjbvb/uz-Cyrl/amp"`;

exports[`AMP uzbek mediaAssetPage SEO Canonical link 1`] = `"http://localhost:7080/uzbek/world-23053613"`;

exports[`AMP uzbek mediaAssetPage SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP uzbek mediaAssetPage SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP uzbek mediaAssetPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP uzbek mediaAssetPage SEO Heading level 1 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston"`;

exports[`AMP uzbek mediaAssetPage SEO Lang attribute 1`] = `"uz-Cyrl"`;

exports[`AMP uzbek mediaAssetPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "about": Array [
    Object {
      "@type": "Thing",
      "name": "Environment",
      "sameAs": Array [
        "http://dbpedia.org/resource/Natural_environment",
      ],
    },
    Object {
      "@type": "Place",
      "name": "Bangladesh",
    },
  ],
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
      "width": 1024,
    },
    "name": "BBC News O'zbek",
    "noBylinesPolicy": "https://www.bbc.com/uzbek/institutional-50220995#authorexpertise",
  },
  "dateModified": "2016-06-22T11:12:52.000Z",
  "datePublished": "2016-06-17T14:24:55.000Z",
  "headline": "Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/uzbek/world-23053613",
    "@type": "WebPage",
    "name": "Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
      "width": 1024,
    },
    "name": "BBC News O'zbek",
    "publishingPrinciples": "https://www.bbc.com/uzbek/institutional-50220995",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
  "url": "http://localhost:7080/uzbek/world-23053613",
}
`;

exports[`AMP uzbek mediaAssetPage SEO OG description 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston"`;

exports[`AMP uzbek mediaAssetPage SEO OG image 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_uzbek/B6D9/test/_63490864_p0140swc.jpg"`;

exports[`AMP uzbek mediaAssetPage SEO OG image alt 1`] = `"Америкада 2 яшар болани тимсоҳ тортиб кетди"`;

exports[`AMP uzbek mediaAssetPage SEO OG locale 1`] = `"uz-UZ"`;

exports[`AMP uzbek mediaAssetPage SEO OG site name 1`] = `"BBC News O'zbek"`;

exports[`AMP uzbek mediaAssetPage SEO OG title 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston - BBC News O'zbek"`;

exports[`AMP uzbek mediaAssetPage SEO OG type 1`] = `"article"`;

exports[`AMP uzbek mediaAssetPage SEO OG url 1`] = `"http://localhost:7080/uzbek/world-23053613"`;

exports[`AMP uzbek mediaAssetPage SEO Page title 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston - BBC News O'zbek"`;

exports[`AMP uzbek mediaAssetPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter creator 1`] = `"@bbcuzbek"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter description 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter image alt 1`] = `"Америкада 2 яшар болани тимсоҳ тортиб кетди"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter image src 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_uzbek/B6D9/test/_63490864_p0140swc.jpg"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter site 1`] = `"@bbcuzbek"`;

exports[`AMP uzbek mediaAssetPage SEO Twitter title 1`] = `"Ўзбекистон, Коронавирус: Мирзиёев Хитой усулини қўлламоқчи - O\`zbekiston - BBC News O'zbek"`;
